### PR TITLE
Update with 6.7.4

### PR DESCRIPTION
**Pull request by abapat**

### CODE REVIEW

**Review by nschmeller — CHANGES_REQUESTED** [review]
Can you remove the changes that remove CAA config?

File: lacework-agent/templates/configmap.yaml [hunk 1]
```diff
@@ -36,13 +36,6 @@ data:
        },
        {{- end }}
      {{- end }}
-     {{- with .Values.laceworkConfig.codeaware }}
-       {{- if .enable }}
-        "codeaware": {
-          "enable": "all"
```
Comment (abapat) [inline lacework-agent/templates/configmap.yaml:42]: just double checking here, we will disable CAA on by default in this fleet-wide release? @geremy-condra

Comment (nschmeller) [reply lacework-agent/templates/configmap.yaml:42]: No, this is likely because the CAA changes didn't make it to the release branch.

Comment (abapat) [reply lacework-agent/templates/configmap.yaml:42]: got it, good catch. Will remove

File: lacework-agent/values.schema.json [hunk 2]
```diff
@@ -310,21 +310,6 @@
                         }
                     }
                 },
-                "codeaware": {
```
Comment (abapat) [inline lacework-agent/values.schema.json:313]: probably this too

Comment (nschmeller) [reply lacework-agent/values.schema.json:313]: Correct, and the values.yaml change

**Review by nschmeller — COMMENTED** [review]
Can you also check that the zipped file doesn't have the CAA changes?

**Comment by abapat** [comment]
it does, fixing them now

**Comment by abapat** [comment]
repackaged the tar and confirmed it has the updated changes (keeping codeaware lines)

**Review by nschmeller — APPROVED** [review]
Thanks!!

**Pull request merged**